processing/src/main/java/org/apache/druid/segment/serde/DictionaryEncodedStringIndexSupplier.java

```diff
@@ -175,6 +177,8 @@ public String getValue(int index)
       extends BaseGenericIndexedDictionaryEncodedIndex<ByteBuffer> implements StringValueSetIndex, Utf8ValueSetIndex
   {
     private static final int SIZE_WORTH_CHECKING_MIN = 8;
+    private static final double SORTED_MERGE_RATIO_THRESHOLD = 0.12D;
```

Comment:
Could you add some javadoc to explain why the default threshold is 0.12?

Reply:
We could link the issue that has the benchmarks.